llvm/lib/ProfileData/InstrProfWriter.cpp

```diff
@@ -552,6 +552,29 @@ static Error writeMemProf(
               memprof::MaximumSupportedVersion));
 }
 
+uint64_t InstrProfWriter::writeHeader(const IndexedInstrProf::Header &Header,
+                                      const bool WritePrevVersion,
+                                      ProfOStream &OS) {
+  // Records the offset before writing any fields.
```

Comment:
Use the infinitive form to be consistent with `Reserve the space ...` below.

```suggestion
  // Record the offset before writing any fields.
```